deeptools/bamCompare.py

```diff
@@ -127,13 +127,13 @@ def getOptionalArgs():
                           required=False)
 
     optional.add_argument('--ratio',
-                          help='The default is to output the log2ratio between the '
+                          help='The default is to output the log2ratio of the '
                           'two samples. The reciprocal ratio returns the '
                           'the negative of the inverse of the ratio '
                           'if the ratio is less than 0. The resulting '
                           'values are interpreted as negative fold changes. '
-                          '*NOTE*: Only when --ratio subtract, the options --normalizeTo1x or '
-                          '--normalizeUsingRPKM can be used.',
+                          '*NOTE*: Only with --ratio subtract can --normalizeTo1x or '
```

Comment:
This sounds strange. `--normalizeTo1x or ... can only be used with "--ratio subtract"`